portfoliostyle.css

```diff
@@ -0,0 +1,310 @@
+* {
+	Box-sizing: border-box;
+	font-family: monaco;
+	
+}
+
+div {
+	display: flex;
+}
+
+body {
+	/*background-color: #aaabb8;*/
+	background-image: url("images/snow-3161134_1920.jpg");
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+}
+
+nav {
+	display: flex;
+	background-color: #25274d;
+	color: #ffffff;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	padding: 0 25px 0 25px;
+	width: 100%;
+}
+
+nav button {
+	margin-left: 10px;
+	padding: 10px;
+	border-radius: 10px;
+	background-color: #29648a;
+	color: #ffffff;
+	border-color: #2e9cca;
+}
+
+main {
+	margin: 0 auto;
+	width: 90%;
+	flex-direction: column;
+	display: flex;
+	
+	font-size: 1.5vw;
+}
+
+
+.about {
+	display: flex;
+	flex-direction: column;
+	margin-top: 25px;
+	padding: 25px 25px 25px 25px;
+	background-color: #464866;
+	color: #ffffff;
+	border-radius: 10px;
+}
+
+.about div {
```

Comment:
On inspect it says the values in here are invalid- think its because you have to add another display: flex; ? Not sure if it is working fine without because the browser has guessed that is what you mean or if you can remove it and it still looks good - might be worth checking :)

Reply:
Thanks, I will look into that